vendor-extra/JatsContentBundle/src/ViewConverter/ArticleCategoriesVisitor.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\ViewConverter;
+
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\SimplifiedVisitor;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverterVisitor;
+
+final class ArticleCategoriesVisitor implements ViewConverterVisitor
+{
+    use SimplifiedVisitor;
+
+    protected function doVisit(Element $object, View $view, array &$context = []) : View
+    {
+        // @fixme - just experimental.
+        return $view->withArgument('items',
```

Comment:
This is not getting picked up. What am I missing?

Reply:
Probably https://github.com/libero/browser/pull/33#discussion_r253848369.